src/js/components/campaignpage/CampaignPage.jsx

```diff
@@ -239,8 +239,8 @@ const filterLowDonations = ({charities, campaign, donationTotal, donation4charit
 		if ( ! donationTotal) {
 			return charities;
 		}
-		// default to 1% of total		
-		lowDntn = Money.mul(donationTotal, 0.01);
+		// default to 0	
+		lowDntn = new Money(donationTotal.currencySymbol + "0");
```

Comment:
Is there an example for why 0 over "low"? 

Having a low but non-zero threshold feels sensible to avoid noise from e.g. if someone sets up one charity during sales, then changes it to another.

Reply:
I changed it because multiple times Kati/sales have complained about charities not appearing when they are actually just under the small threshold - I thought it would be better for all charities to appear and then to be changed or hidden later, rather than leave an ambiguous question of whether the charity was just filtered or actually not coming through in the data